wagtail/documents/templates/wagtaildocs/documents/usage.html

```diff
@@ -10,7 +10,7 @@
             <thead>
                 <tr>
                     <th class="title">{% trans "Title" %}</th>
-                    <th>Field</th>
+                    <th>{% trans "Field" %}</th>
```

Comment:
Looks like the valign="top" in this file can be removed too.

Reply:
gah, undid it to make the screenshot and forgot to remove again.
sorted